docs/endpoints/constructorStandings.md

```diff
@@ -0,0 +1,205 @@
+[← Documentation Home](/docs/README.md)
+# Constructor Standings
+
+Gets a season's constructors standings 
+
+**URL** : `/ergast/f1/{YEAR}/constructorstandings/`
+
+[Available Query Parameters](./README.md#query-parameters)
+
+---
+
+## Route Parameters:
+
+### Season \*\*REQUIRED\*\*
+
+**Filters for the constructors standing of a specified season. Year numbers are valid as is 'current' to get the current seasons constructors standings**
+
+`/{season}/` -> ex: `/ergast/f1/2024/constructorstandings/`
+
+**Note**: To utilize the `season` parameter it needs to be the first route after `/ergast/f1/`
+
+---
+
+### Round
+
+**Filters for the constructors standings after a specified round in a specific season. Round numbers 1 -> n races are valid as well as 'last'**
```

Comment:
```suggestion
**Filters for the constructors standings after a specified round in a specific season. Round numbers 1 -> n races are valid as well as `next` and `last`**
```

Reply:
I fixed the formatting for `last`

but How do we feel about how "next" for example, When we run next on the current season it returns an empty StandingsTable.StandingsList like this:

```
{
  "MRData": {
    "xmlns": "",
    "series": "f1",
    "url": "http://api.jolpi.ca/ergast/f1/2024/next/constructorstandings/",
    "limit": "30",
    "offset": "0",
    "total": "0",
    "StandingsTable": {
      "season": "2024",
      "round": "18",
      "StandingsLists": []
    }
  }
}
```

and if you use `next` on a specific past season like 2018 it will just return the standings after the first round of the 2019 season. While technically it is valid as there is not a server error  but I don't know if it is something that makes sense to list a feature?

Happy to add it. Just wanted to present that thought.

Reply:
Yeah you're totally right, `next` will never make sense on the driver/constructor standing endpoints.
So I think you're right about not mentioning it.